packages/datagateway-common/src/homePage/homePage.component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { createMount, createShallow } from '@material-ui/core/test-utils';
+import { HomePageWithoutStyles, HomePageProps } from './homePage.component';
+
+describe('Home page component', () => {
+  let shallow;
+  let mount;
+  let props: HomePageProps;
+
+  beforeEach(() => {
+    shallow = createShallow({ untilSelector: 'div' });
+    mount = createMount();
+
+    props = {
+      classes: {
+        bigImage: 'bigImage-class',
+        howItWorks: 'howItWorks-class',
+        howItWorksTitle: 'howItWorksTitle-class',
+        howItWorksGridItem: 'howItWorksGridItem-class',
+        howItWorksGridItemTitle: 'howItWorksGridItemTitle-class',
+        howItWorksGridItemImage: 'howItWorksGridItemImage-class',
+        howItWorksGridItemCaption: 'howItWorksGridItemCaption-class',
+      },
+    };
+  });
+
+  afterEach(() => {
+    mount.cleanUp();
+  });
+
+  it('homepage renders correctly', () => {
+    const wrapper = shallow(<HomePageWithoutStyles {...props} />);
```

Comment:
You are using `HomePageWithoutStyles` which is why Codecov warns that `createStyles` is not covered. Instead import the whole component by adding `import HomePage from './homePage.component';` at the top and pass the props to it like you are already doing - `const wrapper = shallow(<HomePage {...props} />);`.